build.gradle

```diff
@@ -32,9 +32,9 @@ publishing {
         mavenJava(MavenPublication) {
             from components.java
             pom {
-                name = 'Libra Client SDK'
-                description = 'The official Libra Client SDK for the Java programming language.'
-                url = 'https://github.com/libra/libra-client-sdk-java'
+                name = 'Diem Client SDK'
+                description = 'The official Diem Client SDK for the Java programming language.'
+                url = 'https://github.com/libra/client-sdk-java'
```

Comment:
@JoelMarcey when are we changing the libra repo name? :P

Reply:
Soon, I hope. I think we need to settle a few issues first and then try to transfer the orgs.